### PR TITLE
updating Pod DNS config example to use documentation IP address

### PR DESCRIPTION
**Pull request by reetasingh**
Fix https://github.com/kubernetes/website/issues/39166
Problem:
https://kubernetes.io/docs/concepts/services-networking/dns-pod-service/#pod-dns-config uses 1.2.3.4 as the example DNS server. It's better to use an [IP address reserved for documentation](https://www.rfc-editor.org/rfc/rfc5737.html).

Proposed Solution:
Change the example manifest and web page to use IP address 192.0.2.1 ( from the 192.0.2.0/24 block)

### CODE REVIEW

**Review by kundan2707 — COMMENTED** [review]
LGTM

**Comment by tengqm** [comment]
I don't think this is necessary. `1.2.3.4` reads more like an intented, fake DNS server address.
Changing this to something more realistic may be confusing.

/hold

**Comment by reetasingh** [comment]
> #39166

[sftim](https://github.com/sftim) whats your thought?

**Comment by reetasingh** [comment]
> I don't think this is necessary. `1.2.3.4` reads more like an intented, fake DNS server address. Changing this to something more realistic may be confusing.
> 
> /hold

[sftim](https://github.com/sftim) whats your thought about this?

File: content/en/examples/service/networking/custom-dns.yaml [hunk 1]
```diff
@@ -10,7 +10,7 @@ spec:
   dnsPolicy: "None"
   dnsConfig:
     nameservers:
-      - 1.2.3.4
+      - 192.0.2.1
```
Comment (sftim) [inline content/en/examples/service/networking/custom-dns.yaml:13]: To address @tengqm's [reservation](https://github.com/kubernetes/website/pull/39169#issuecomment-1409944922), how about:
```suggestion
      - 192.0.2.1 # this is an example
```

_or_ use an AS112 IP address, such as 10.42.0.1?

```suggestion
      - 10.42.0.1 # this is an example
```

How about either of those approaches? I notice that the IPv6 example is already using a documentation IP address.

Comment (tengqm) [reply content/en/examples/service/networking/custom-dns.yaml:13]: Adding a comment is good and clear enough.
Thanks.

File: content/en/docs/concepts/services-networking/dns-pod-service.md [hunk 2]
```diff
@@ -306,7 +306,7 @@ When the Pod above is created, the container `test` gets the following contents
 in its `/etc/resolv.conf` file:
 
 ```
-nameserver 1.2.3.4
+nameserver 192.0.2.1
```
Comment (tengqm) [inline content/en/docs/concepts/services-networking/dns-pod-service.md:309]: please add a comment like `# 192.0.2.1 is an example IP`.

Comment (sftim) [reply content/en/docs/concepts/services-networking/dns-pod-service.md:309]: >       Lines that contain a semicolon (;) or hash character (#) in the
>       first column are treated as comments.

For `resolv.conf`, that comment does need to be on a line by itself.

**Comment by reylejano** [comment]
@reetasingh ,
Can you push an empty commit to restart the tests
On your `reetasingh:reetas_update_dns` branch, you can do
`git commit --allow-empty -m "empty commit"`
`git push origin reetas_update_dns`

**Comment by reetasingh** [comment]
> @reetasingh , Can you push an empty commit to restart the tests On your `reetasingh:reetas_update_dns` branch, you can do `git commit --allow-empty -m "empty commit"` `git push origin reetas_update_dns`

Hi @reylejano pushed an empty commit

**Pull request merged**